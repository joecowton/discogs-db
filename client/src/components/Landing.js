import React from 'react';

const Landing = () => {
  return(
    <div class="jumbotron center">
      <h1>Videos!</h1>

      <p><a class="btn btn-primary btn-lg" href="/auth/google" role="button">Log in</a></p>
    </div>
  )
}

export default Landing;
