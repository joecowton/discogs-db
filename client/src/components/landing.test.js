import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

describe('Landing', () => {
    it('should render the component', () => {
        const wrapper = shallow(<Landing />);
        expect(wrapper).toMatchSnapshot();
    });
});
