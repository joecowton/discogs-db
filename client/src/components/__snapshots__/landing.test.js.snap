// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render the component 1`] = `
<div
  className="jumbotron center"
>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <img
      align="middle"
      alt="main"
      src="http://livitysound.com/site/wp-content/uploads/2015/09/livitysound-logo.jpg"
    />
  </div>
</div>
`;
